pdns/dnsdistdist/docs/reference/config.rst

```diff
@@ -1614,6 +1614,14 @@ faster than the existing rules.
 
     :param list netmasks: A :class:`NetmaskGroup` object, or a netmask or list of netmasks as strings, like for example "192.0.2.1/24"
 
+  .. method:: DynBlockRulesGroup:removeRange(netmasks)
+
+    .. versionadded:: 1.9.0
```

Comment:
1.8.1 ?

Reply:
1.8.3, if I'm not mistaken, but we will have to fix this in master since the documentation on stable branches is not used.

Reply:
I meant 1.8.x :/ sorry.

Reply:
https://github.com/PowerDNS/pdns/pull/13622